api-guidelines/global/core-principles/documentation.md

```diff
@@ -1,57 +1,39 @@
-- The layout supports accordingly, e.g. with syntax highlighting or multi-column layout.
-  :::
+- The layout supports usability accordingly, e.g. with syntax highlighting or multi-column layout.
 
-::: accordion Code examples
-**Default values**  
-Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+## Code examples
 
-**Code examples and schemas**  
-Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Every endpoint documentation comes with examples and also provides schemas listing the available attributes with explanatory text.
+- Request and response examples that belong to every endpoint documentation have meaningful default values that follow a specific story.
+- Code examples and explanatory text are visually separated.
+- API endpoints and examples are shown in context.
+- API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
 
-**Automated API reference documentation**  
-API reference documentation is created and published automatically, see [MUST provide API specification using OpenAPI](../../rest/contract/openapi/rules/must-provide-api-specification-using-openapi-for-rest-apis.md).
-:::
+## Context and walkthroughs
 
-::: accordion Consistency and accessibility
+- We give context for API usage such as information about HTTP methods, API operations, request and response headers, versioning, pagination, or errors.
+- With step-by-step walkthroughs we'll cover specific functionality that can be implemented with the API.
+- We cater for experienced users that already have an idea of the endpoints to be used, and at the same time provide the respective structure and information for inexperienced users who need a thorough introduction and/or example use cases.
+- We provide an overview of the most likely tasks to perform with the API.
 
-- Documentation is provided in American English (EN-US), as US spelling has become the standard in APIs. Also endpoints, properties, and default values are provided in American English.
-- Users can search the documentation.
-- The sidebar is available at all times so that the documentation can be easily browsed. All the endpoints can be accessed via the sidebar.
-- Code examples and explanatory text are visually separated.
-  :::
+## Up-to-dateness
 
-::: accordion Efficient and refined
-The API is well-documented and up-to-date.  
-Users can flawlessly integrate and are informed about recent changes via a revision history.
-:::
-::::
+- The API is well-documented and up-to-date.  
+- Users can flawlessly integrate and are informed about recent changes.
```

Comment:
At the moment, users can only inform themselves about recent changes. There is no automation to subscribe to changes of one specific endpoint.

Reply:
This relates to the changelog. I'll rewrite this.